app/src/main/res/layout-land/section_title.xml

```diff
@@ -27,9 +27,9 @@
       android:id="@+id/section_title_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="96dp"
+      android:layout_marginStart="72dp"
       android:layout_marginTop="28dp"
```

Comment:
According to the mocks, this should be `32dp`  
Also on line 22 of the same file, `android:layout_marginTop="28dp"` should be `android:layout_marginTop="32dp"`

Reply:
Done.